jenkins/custom-ci/generic/cortx-rgw-integration.groovy

```diff
@@ -41,7 +47,15 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
 
                 sh encoding: 'UTF-8', label: 'cortx-provisioner', script: '''
-                bash ./jenkins/build.sh -v 2.0.0 -b ${CUSTOM_CI_BUILD_ID}
+                    if [ "${ENABLE_ADDB_PLUGIN}" == "yes" ]; then
+                        pushd ../
```

Comment:
I will Add the input parameter for the branch name and branch URL and export them here post successful testing.